modules/Kconfig.tfm

```diff
@@ -6,17 +6,49 @@
 
 
 config TFM_BOARD
```

Comment:
why not use TFM_BOARD as a choice with an UNSUPPORTED_BOARD as default.

Reply:
> why not use TFM_BOARD as a choice with an UNSUPPORTED_BOARD as default.

Because a choice cannot be promptless.
This is not something a user should configure but something a board implementation decides.